devices/universal_electronics_inc.js

```diff
@@ -11,7 +11,7 @@ module.exports = [
         zigbeeModel: ['URC4470BC0-X-R'],
         model: 'XHS1-UE',
         vendor: 'Universal Electronics Inc',
-        description: 'Wireless Digital Pet Resistant PIR Detector',
+        description: 'Wireless digital pet resistant PIR detector',
```

Comment:
fwiw, this was straight from the manual: https://fccid.io/MG3-H74470/User-Manual/User-Manual-4666852